### PR TITLE
Add create tag, assign tag, unassign tag features to UG

### PR DESCRIPTION
**Pull request by glennljw**
Fixes part of #84

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -192,6 +192,55 @@ Examples:
...
+Format: `assign INDEX TAGNAME`
+
+* Assigns a `TAG` with a given `TAGNAME` to a contact at the specified `INDEX`
+* The `TAG` given by the `TAGNAME` must be created first. 
+* The `TAGNAME` is case-insensitive.
+* The index refers to the index number shown in the displayed contact list.
+* The index **must be a positive integer** 1, 2, 3, ...​
+* The contact should have **at most one** `TAG` with a given unique `TAGNAME`.
+* The contact assigned to the given `TAGNAME` will not be assigned to the same `TAGNAME` again.
+e.g. assigning the tag `friends` to Alice at index 1 will not allow `Friends` to be assigned to the same contact.
```
Comment (weijuey) [inline docs/UserGuide.md:221]: Given the constraints stated under `tag` for creating tags, aren't these scenarios impossible anyway? 🤔  Though I think it's still good to mention tagging a contact with a tag he already has will do nothing.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -192,6 +192,55 @@ Examples:
...
+
+Examples:
+
+* `assign 1 Friends` assigns a tag `Friends` to the contact at index `1`
+
+### Unassigning a tag to a contact : `unassign`
+
+Unassigns a created tag to a contact.
```
Comment (weijuey) [inline docs/UserGuide.md:229]: "from a contact" might be more grammatically correct, I think? 😄

**Review by weijuey — CHANGES_REQUESTED** [review]
Small nits but otherwise, lgtm! Pretty smooth to read 😃

File: docs/UserGuide.md (hunk 2 above)
Comment (glennljw) [reply docs/UserGuide.md:229]: Yep my bad!

**Review by weijuey — APPROVED** [review]

**Pull request merged**